mysql-test/main/failed_auth_unixsocket.test

```diff
@@ -20,7 +20,7 @@ let $replace=Access denied for user '$USER';
 connect (fail,localhost,$USER);
 --enable_query_log
 
---replace_result $replace "Access denied for user 'USER'"
+--replace_result $USER USER
```

Comment:
generally it's not a good idea, we've have cases before when a user name (we cannot control the value of $USER) indiscriminately matched some random part of the message, so more specific replacements are preferred. What about
```
--replace_result $replace "Access denied for user 'USER'" "change_user $USER" "change_user USER"
```
will that work?